plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -23,4 +23,20 @@
     <persistent value="true" />
   </extension>
   
+  <extension 
+      name="My Validator" 
+      point="org.eclipse.wst.validation.validatorV2" 
+      id="com.google.cloud.tools.eclipse.appengine.validation.MyValidator">
```

Comment:
If `name` and `id` are required, perhaps _App Engine Descriptor Validator_ might be better than _My Validator_